src/main/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentDisposer.java

```diff
@@ -0,0 +1,156 @@
+package com.cloudbees.jenkins.plugins.sshagent;
+
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.Util;
+import hudson.model.TaskListener;
+import hudson.model.Run;
+import hudson.security.ACL;
+import hudson.util.Secret;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+
+import jenkins.model.Jenkins;
+import jenkins.tasks.SimpleBuildWrapper.Context;
+import jenkins.tasks.SimpleBuildWrapper.Disposer;
+
+import org.apache.commons.lang.StringUtils;
+
+import com.cloudbees.jenkins.plugins.sshcredentials.SSHUser;
+import com.cloudbees.jenkins.plugins.sshcredentials.SSHUserPrivateKey;
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import com.cloudbees.plugins.credentials.common.StandardUsernameCredentials;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * TODO: Add description.
+ */
+public class SSHAgentDisposer extends Disposer {
+
+    /**
+     * Serial Version UID.
+     */
+    private static final long serialVersionUID = -5705766598445563171L;
```

Comment:
@jglick @kohsuke I've configured my IDE to generate this attribute automatically.